src/main/webapp/resources/views/user/UserPage.html

```diff
@@ -0,0 +1,338 @@
+                                                </div>
+                                                <div class="col-xs-6">
+                                                    <div class="right-box">Still needed</div>
+                                                    <div class="right-box"><span class="fa fa-dollar"></span>9999</div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <div class="col-xs-12">
+        <div class="bs-calltoaction bs-calltoaction-success">
+            <div class="row">
+                <div class="col-md-9 cta-contents">
+                    <a href="#!/projects"><h1 class="cta-title">Projects</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#!/projects">My Projects</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addProject">
+                        <fieldset>
+                            <div class="form-group col-md-12">
+                                <div>
+                                    <input id="ProjectName" name="ProjectName" type="text"
+                                           placeholder="Enter project name"
+                                           class="form-control input-md input-text">
+                                </div>
+                                <div class="cta-button">
+                                    <button type="submit" class="btn btn-lg btn-block btn-edit">Add Project</button>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </form>
+                </div>
+            </div>
+        </div>
+        <div class="bs-calltoaction bs-calltoaction-success">
+            <div class="row">
+                <div class="col-md-9 cta-contents">
+                    <a href="#"><h1 class="cta-title">Interests</h1></a>
+                    <div class="cta-desc">
+                        <p><a href="#!/interests">My Interests</a></p>
+                        <p>Describe the action here.</p>
+                        <p>Describe the action here.</p>
+                    </div>
+                </div>
+                <div class="col-md-3 form-field">
+                    <form class="form-horizontal" action="" method="" id="addInterest">
+                        <fieldset>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a <legend> tag to this fieldset. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)